subgraphs/aggregator/schema.graphql

```diff
@@ -0,0 +1,212 @@
+type Aggregator @entity {
+  " { Currency } "
+  id: ID!
+
+  " Trade currency "
+  currency: Bytes!
+
+  " Total number of unique users "
+  users: BigInt!
+
+  " Total number of collections traded "
+  collections: BigInt!
+
+  " Total number of transactions "
+  transactions: BigInt!
+
+  " Total volume "
+  volume: BigDecimal!
+
+  " Aggregator daily data "
+  dailyData: [AggregatorDailyData!]! @derivedFrom(field: "aggregator")
+}
+
+type AggregatorDailyData @entity {
+  " { Currency }-{ Date } "
+  id: ID!
+
+  " Trade currency "
+  currency: Bytes!
+
+  " Pointer to Aggregator "
+  aggregator: Aggregator!
+
+  " Date "
+  date: BigInt!
+
+  " Total number of unique users "
+  users: BigInt!
+
+  " Total number of collections traded "
+  collections: BigInt!
+
+  " Total number of transactions "
+  transactions: BigInt!
+
+  " Total volume "
+  volume: BigDecimal!
+}
+
+type Collection @entity {
+  " { Address }-{ Currency } "
+  id: ID!
+
+  " Trade currency "
+  currency: Bytes!
+
+  " Total number of transactions "
+  transactions: BigInt!
+
+  " Total volume "
+  volume: BigDecimal!
+
+  " Collection daily data "
+  dailyData: [CollectionDailyData!]! @derivedFrom(field: "collection")
+}
+
+type CollectionDailyData @entity {
+  " { Address }-{ Currency }-{ Date } "
+  id: ID!
```

Comment:
This should cause conflict since 2 collections would have the same id. It should incorporate the collection address in it